src/status_im2/contexts/communities/actions/token_gating/view.cljs

```diff
@@ -0,0 +1,23 @@
+(ns status-im2.contexts.communities.actions.token-gating.view
+  (:require [quo2.core :as quo]
+            [utils.i18n :as i18n]
+            [utils.re-frame :as rf]
+            [react-native.core :as rn]))
+
+(defn token-requirements
+  [id]
+  (rf/dispatch [:communities/check-permissions-to-join-community id])
```

Comment:
is it not better to use `on-component-did-mount` etc?

Reply:
it's simplest one, but works the same